h/static/scripts/annotation-ui.js

```diff
@@ -290,12 +280,13 @@ module.exports = function (settings) {
     },
 
     /** De-select an annotation. */
-    removeSelectedAnnotation: function (annotation) {
+    removeSelectedAnnotation: function (id) {
```

Comment:
Since the function only uses `annotation.id` it can just accept `id` as arg, doesn't need the rest of `annotation`.